lib/private/Template/JSConfigHelper.php

```diff
@@ -201,6 +202,7 @@ public function getConfig() {
 			'sharing.maxAutocompleteResults' => max(0, $this->config->getSystemValueInt('sharing.maxAutocompleteResults', Constants::SHARING_MAX_AUTOCOMPLETE_RESULTS_DEFAULT)),
 			'sharing.minSearchStringLength' => $this->config->getSystemValueInt('sharing.minSearchStringLength', 0),
 			'blacklist_files_regex' => \OCP\Files\FileInfo::BLACKLIST_FILES_REGEX,
+			'loglevel' => $this->config->getSystemValue('loglevel', ILogger::WARN),
```

Comment:
this *could* be a sensitive value. How about we introduce a `loglevel_browser` that falls back to `loglevel`?

cc @nickvergessen

Reply:
I can not imagine an attack using this off-hand, but nevertheless this sounds reasonable, so if that is the preferred way I can change the pull request.

Reply:
> How about we introduce a loglevel_browser that falls back to loglevel?

I think that sounds like a good middle ground:
```suggestion
			'loglevel' => $this->config->getSystemValueInt('loglevel_browser', $this->config->getSystemValueInt('loglevel', ILogger::WARN)),
```